docs/json/radarr/cf/anime-dual-audio.json

```
@@ -9,7 +9,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "dual[ ._-]?audio|[\\[(]dual[\\])]|(JA|ZH)\\+EN|EN\\+(JA|ZH)"
+        "value": "dual[ ._-]?audio|[\\[(]dual[\\])]|(JA|ZH|KO)\\+EN|EN\\+(JA|ZH|KO)"
```

```suggestion
        "value": "dual[ ._-]?audio|[([]dual[])]|(JA|ZH|KO)\\+EN|EN\\+(JA|ZH|KO)"
```
this should work as well without causing issues with mkdocs